package.json

```diff
@@ -3,6 +3,10 @@
   "version": "0.2.0",
   "description": "Component library based on the Quantum Design System",
   "main": "dist/index.js",
+  "exports": {
+    ".": "./dist/index.js",
+    "./plugin": "./dist/plugin.js"
```

Comment:
Is it correct that the plugin is now the primary method for using this library?
If so, should the plugin be the default import and index.js secondary?

i.e.

```
  "exports": {
    ".": "./dist/plugin.js",
    "./components": "./dist/index.js" (or "./lib": ... ?) 
   }
```